tests/components/google_assistant/test_trait.py

```diff
@@ -816,6 +821,340 @@
     assert len(calls) == 2
 
 
+async def test_arm_disarm_arm_away(hass):
+    """Test ArmDisarm trait Arming support for alarm_control_panel domain."""
+    assert helpers.get_google_type(alarm_control_panel.DOMAIN, None) is not None
+    assert trait.ArmDisArmTrait.supported(alarm_control_panel.DOMAIN, 0, None)
+    assert trait.ArmDisArmTrait.might_2fa(alarm_control_panel.DOMAIN, 0, None)
+
+    trt = trait.ArmDisArmTrait(
+        hass,
+        State(
+            "alarm_control_panel.alarm",
+            STATE_ALARM_ARMED_AWAY,
+            {alarm_control_panel.ATTR_CODE_ARM_REQUIRED: True},
+        ),
+        PIN_CONFIG,
+    )
+    assert trt.sync_attributes() == {
+        "availableArmLevels": {
+            "levels": [
+                {
+                    "level_name": "armed_home",
+                    "level_values": [
+                        {"level_synonym": ["armed home", "home"], "lang": "en"}
+                    ],
+                },
+                {
+                    "level_name": "armed_away",
+                    "level_values": [
+                        {"level_synonym": ["armed away", "away"], "lang": "en"}
+                    ],
+                },
+                {
+                    "level_name": "armed_night",
+                    "level_values": [
+                        {"level_synonym": ["armed night", "night"], "lang": "en"}
+                    ],
+                },
+                {
+                    "level_name": "armed_custom_bypass",
+                    "level_values": [
+                        {
+                            "level_synonym": ["armed custom bypass", "custom"],
+                            "lang": "en",
+                        }
+                    ],
+                },
+                {
+                    "level_name": "triggered",
+                    "level_values": [
+                        {"level_synonym": ["triggered", "triggered"], "lang": "en"}
```

Comment:
these synonyms are duplicate?

Reply:
The reason is this a duplicate is because for the Trigger state it is one word. and using the logic I did for the synonyms the result will be the same twice.
The alternative would be to hard code the synonym. what do you think?

Reply:
let's create the synonyms above this dict statement and add an if-check for triggered there.